packages/commonwealth/client/scripts/models/Thread.ts

```diff
@@ -375,8 +375,7 @@ export class Thread implements IUniqueId {
     this.identifier = `${id}`;
     this.createdAt = moment(created_at);
     this.updatedAt = moment(updated_at);
-    // @ts-expect-error StrictNullChecks
-    this.topic = topic?.id ? new Topic({ ...(topic || {}) } as any) : null;
+    this.topic = { ...topic };
```

Comment:
Unrelated to PR: The topic can be `null/undefined` if the admin deletes the thread topic. I tested but it doesn't seem to break anything other than the author info in the skeleton load state in `<ThreadCard/>` on the `/discussions` page and an undefined `.id` reference in `<AdminActions/>` both of which are also on master.

Reply:
Can you address this in a separate ticket please?